module/zstd/zfs_zstd.c

```diff
@@ -135,7 +188,7 @@ static void zstd_free(void *opaque, void *ptr);
 static const ZSTD_customMem zstd_malloc = {
 	zstd_alloc,
 	zstd_free,
-	NULL,
+		NULL,
```

Comment:
nit: stray extra tab

Reply:
I'm...not sure how that even happened, it's not like I was editing anywhere near there.

Fixed.